lib/runner.js

```diff
@@ -633,6 +634,16 @@ Runner.prototype.runSuite = function(suite, fn) {
 
   debug('run suite %s', suite.fullTitle());
 
+  if (!total && this.forbidOnly && hasOnly(suite)) {
+    suite._onlySuites.forEach(function(onlySuite) {
```

Comment:
I really don't love this... it seems like a pretty gnarly hack. 😐 	 I'll see if I can find a more elegant alternate implementation.